lib/src/features/product/provider/pay_app_provider.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:io';
```

Comment:
Please create a controller class using (Async)Notifier for business logic for one UI page.
In this case, there is a ProductPage. Please create a ProductController for storing business logic.
And put ~controller.dart and ~page.dart into the same directory.
```
- product/presentation/product_page.dart
- product/presentation/product_controller.dart
```